VirtoCommerce.CatalogModule.Web/Scripts/directives/property2.js

```diff
@@ -130,6 +131,13 @@ angular.module('virtoCommerce.catalogModule')
                 }
             };
 
+            scope.isLanguageVisible = function (language) {
```

Comment:
I think you need to do additional null checks for  `scope.hiddenLanguages` here

Reply:
Done